h2o-algos/src/main/java/hex/tree/isoforextended/ExtendedIsolationForest.java

```diff
@@ -237,4 +258,50 @@ public TwoDimTable createModelSummaryTable() {
         return table;
     }
 
+    protected TwoDimTable createScoringHistoryTable() {
+        List<String> colHeaders = new ArrayList<>();
+        List<String> colTypes = new ArrayList<>();
+        List<String> colFormat = new ArrayList<>();
+        colHeaders.add("Timestamp"); colTypes.add("string"); colFormat.add("%s");
+        colHeaders.add("Duration"); colTypes.add("string"); colFormat.add("%s");
+        colHeaders.add("Number of Trees"); colTypes.add("long"); colFormat.add("%d");
+        colHeaders.add("Mean Tree Path Length"); colTypes.add("double"); colFormat.add("%.5f");
+        colHeaders.add("Mean Anomaly Score"); colTypes.add("double"); colFormat.add("%.5f");
```

Comment:
In different PR (https://github.com/h2oai/h2o-3/pull/6035) you changed "Mean Anomaly Score" to "Mean Score", so it is correct here?

```suggestion
        colHeaders.add("Mean Score"); colTypes.add("double"); colFormat.add("%.5f");
```

Reply:
This is ok for me. I changed it only in the Metrics not to mess up together Anomaly Score <–> Score. In metrics Score is any raw number that is input to the Normalized Anomaly Score <-> some number between 0 and 1. 

In this place I call number between 0 and 1 anomaly score which is correct. In the MetricsAnomaly it was not correct because we called path from root to leaf as a Anomaly score.